WcaOnRails/app/mailers/delegate_status_change_mailer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class DelegateStatusChangeMailer < ApplicationMailer
+  def notify_board_and_wqac_of_delegate_status_change(user_whose_delegate_status_changes, user_senior_delegate)
```

Comment:
I think `user_senior_delegate` is a confusing variable name here. Is this a user who is a senior delegate? Is this the users's senior delegate? Based on how you're using it later on, it seems to me like it should be called `changer` or even `user_who_made_the_change`.

Reply:
user_who_made_the_change sounds better, I will change this.